h/static/scripts/auth.coffee

```diff
@@ -70,11 +70,9 @@ class Auth
     # has failed and if yes, it sets the user value to null. (Otherwise the
     # onlogin method would set it to userId)
     onready = =>
-      if @user is undefined and not _checkingToken
-        @user = null
+      if auth.user is undefined and not _checkingToken
+        auth.user = null
 
     identity.watch {onlogin, onlogout, onready}
-
-
-angular.module('h')
-.service('auth', Auth)
+    auth
```

Comment:
Could you make this return explicit? I can't stand CoffeeScript's implicit return. It's so easy to miss this line but the entire service will be screwed if it's missing.